packages/Upphandling/components/Tenders.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react'
+import { ScrollView, View } from 'react-native'
+import {
+  Button,
+  StyleService,
+  Text,
+} from '@ui-kitten/components'
+import { useTenders } from '../hooks/useTenders'
+import { ImageOverlay } from './ImageOverlay'
+import moment from 'moment'
+import { Tag } from './Tag'
+
+const Tender = ({ tender, onPress }) => {
+  const {
+    id,
+    description,
+    issues,
+    geography,
+    services,
+    technologies,
+    evaluationCriteria,
+    startDate,
+  } = tender
+
+  return (
+    <ImageOverlay
+      onPress={onPress}
+      style={styles.tender}
+      imageBackgroundProps={styles.image}
+      source={require('../assets/background.png')}
+    >
+      <View style={styles.tenderHeader}>
+        <Text category={'h6'}>{description}</Text>
+        <Text category={'c1'}>
+          {moment(startDate).calendar()}
```

Comment:
[On moment's website they state the following:](https://momentjs.com/)
> Considering using Moment in your project?
There may be better modern alternatives.
For more details and recommendations,
please [see Project Status in the docs](https://momentjs.com/docs/#/-project-status/).


So it seems to me it's kind of deprecated nowadays and we should consider if we should be using it.

A library that supports this kind of relative dates that isn't moment would be https://sugarjs.com/dates/#/Formatting